app/org/sagebionetworks/bridge/models/studies/MimeType.java

```diff
@@ -0,0 +1,17 @@
+package org.sagebionetworks.bridge.models.studies;
+
+public enum MimeType {
```

Comment:
Found 2 alternatives: http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/http/MediaType.html and http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/net/MediaType.html.